Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -518,25 +516,55 @@ - (void)loadVungleNativeAdForParameters:(id<MAAdapterResponseParameters>)paramet
     return clickableViews;
 }
 
-- (BannerSize)adSizeFromAdFormat:(MAAdFormat *)adFormat
+- (VungleAdSize *)adSizeFromAdFormat:(MAAdFormat *)adFormat
+                          parameters:(id<MAAdapterParameters>)parameters
 {
-    if ( adFormat == MAAdFormat.banner )
-    {
-        return BannerSizeRegular;
-    }
-    else if ( adFormat == MAAdFormat.leader )
+    BOOL isAdaptiveBanner = [parameters.localExtraParameters al_boolForKey: @"adaptive_banner"];
+    NSNumber *customWidth = [parameters.localExtraParameters al_numberForKey: @"adaptive_banner_width"];
+    NSNumber *customHight = [parameters.localExtraParameters al_numberForKey: @"adaptive_banner_height"];
+    
+    if ( isAdaptiveBanner )
     {
-        return BannerSizeLeaderboard;
+        return [VungleAdSize VungleAdSizeWithWidth:[self adaptiveBannerWidthFromParameters: parameters]];
```

Comment:
we r not gonna support this api in phase 1. so u should remove this in this pr. 
next phase we can add it.

Reply:
Changed to use `VungleAdSizeFromCGSize` instead.